aesara/sparse/basic.py

```diff
@@ -413,6 +416,12 @@ def __repr__(self):
 SparseType.Variable = SparseVariable
 SparseType.Constant = SparseConstant
 
+# TODO:
+# @_as_tensor_variable.register(SparseVariable)
+# @_as_tensor_variable.register(SparseConstant)
+# def _as_tensor_sparse(x, name, ndim):
+#     return x
+
```

Comment:
This TODO could also use an explanation or link to a Github issue

Reply:
I was waiting to merge #302.